### ISSUE TITLE
conf location

### ISSUE THREAD

**Issue by BadCo-NZ** [ibracorp/authelia#2]
Hi, I am using swag and was wondering where Authelia Portal.conf and Protected Endpoint.conf should be placed?

Do they go in the Authelia container or in proxy.conf in the nginx/swag container?

Sorry if it is in the documentation, but I couldn't see it.

**Comment by BadCo-NZ** [comment]
This post by linuxserver.io helped fill the gaps with the differences between NPM and SWAG, still need to fix something but that will wait for tomorrow.

https://blog.linuxserver.io/2020/08/26/setting-up-authelia/

**Comment by ibracorp** [comment]
Hi there and thanks for the email. Sorry about the late reply.

I'm glad you found some extra resources to help out. To answer your
original question, the conf files are provided for people following this
particular documentation with NPM.

In several steps it describes where to copy the information from the conf
files to places such as NPM Advanced tab on a proxy host that's been setup,
one for the endpoint and one for Authelia itself.

Because SWAG/LetsEncrypt already has the template baked in to its conf
templates it's slot easier in that respect. As for NPM, this templates
don't exist so I had to combine the code in them to work with NPM.

So in summary, the official Authelia documentation actually describes how
to get it working with NGINX/LetsEncrypt setup, in a more condensed version
as it refers to seperated files instead one long conf.

Good luck and if you need more help I'm happy to lend a hand.

Thanks!
